op-chain-ops/genesis/genesis.go

```diff
@@ -91,35 +92,22 @@ func NewL2Genesis(config *DeployConfig, l1StartHeader *types.Header) (*core.Gene
 		difficulty = newHexBig(0)
 	}
 
-	extraData := config.L2GenesisBlockExtraData
-	if extraData == nil {
-		// L2GenesisBlockExtraData is optional, so use a default value when nil
-		extraData = BedrockTransitionBlockExtraData
-	}
-	// Ensure that the extradata is valid
-	if size := len(extraData); size > 32 {
-		return nil, fmt.Errorf("transition block extradata too long: %d", size)
-	}
```

Comment:
The way that `superchain-registry` does this is by checking out a particular commit and running a command. We want to reduce the number of footguns for reproducible builds, we are essentially taking over extradata as part of consensus

Reply:
We could only set the extra data if pre-Holocene, otherwise set it correctly and ignore the field. But if the SCR works on old commits of genesis generation anyways, it's good to remove this.

If others scream that we need to add this functionality back, we can do it.